editor/import/post_import_plugin_skeleton_rest_fixer.cpp

```diff
@@ -109,6 +102,43 @@ void PostImportPluginSkeletonRestFixer::internal_process(InternalImportCategory
 				}
 				pr = pr->get_parent();
 			}
+			global_transform.origin = Vector3(); // Translation by a Node is not a bone animation, so the retargeted model should be at the origin.
+		}
+
+		// Calc IBM difference.
+		LocalVector<Vector<Transform3D>> ibm_diffs;
+		{
+			TypedArray<Node> nodes = p_base_scene->find_children("*", "ImporterMeshInstance3D");
+			while (nodes.size()) {
+				ImporterMeshInstance3D *mi = Object::cast_to<ImporterMeshInstance3D>(nodes.pop_back());
+				Ref<Skin> skin = mi->get_skin();
+				if (skin.is_valid()) {
+					Node *node = mi->get_node(mi->get_skeleton_path());
```

Comment:
I prefer early return to avoid conditional nesting 5 layers deep.

if (!condition) {
   continue;
}